workflow/cron/operator_test.go

```diff
@@ -339,7 +339,7 @@ var forbidMissedSchedule = `apiVersion: argoproj.io/v1alpha1
 kind: CronWorkflow
 metadata:
   annotations:
-    cronworkflows.argoproj.io/last-used-schedule: CRON_TZ=America/Los_Angeles 0-36/1
+    cronworkflow.argoproj.io/last-used-schedule: CRON_TZ=America/Los_Angeles 0-36/1
```

Comment:
Would this be a breaking change if users are already using it?

Reply:
Formatting issue I think